### DISCUSSION TITLE
Search icon instead of search filed

### DISCUSSION THREAD

**Discussion by balrogmedia** [bootscore/bootscore#290] [Q&A] [answered]
Hello,

is it possible to display on desktop a search icon (like on mobile) on the navbar, instead of the search field?

thank you

**Comment by crftwrk** [comment]
Of course, this is possible. Open `header.php` and do following changes:

#### Find

```html
<!-- Searchform Large -->
<div class="d-none d-lg-block ms-1 ms-md-2 top-nav-search-lg">
  <?php if (is_active_sidebar('top-nav-search')) : ?>
    <div>
      <?php dynamic_sidebar('top-nav-search'); ?>
    </div>
  <?php endif; ?>
</div>
```

Delete snippet

#### Find

```html
<!-- Search Toggler Mobile -->
<button class="btn btn-outline-secondary d-lg-none ms-1 ms-md-2 top-nav-search-md" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-search" aria-expanded="false" aria-controls="collapse-search">
  <i class="fa-solid fa-magnifying-glass"></i><span class="visually-hidden-focusable">Search</span>
</button>
```

Remove class `d-lg-none`

#### Find

```html
<!-- Top Nav Search Mobile Collapse -->
<div class="collapse container d-lg-none" id="collapse-search">
  <?php if (is_active_sidebar('top-nav-search')) : ?>
    <div class="mb-2">
      <?php dynamic_sidebar('top-nav-search'); ?>
    </div>
  <?php endif; ?>
</div>
```

Remove class `d-lg-none`

Does this help?

**Comment by balrogmedia** [comment]
helps a lot! wonderful thanks!

**Comment by crftwrk** [comment, marked as answer]
👍 You're welcome. Plase mark as answered ;-)

**Comment by balrogmedia** [comment]
Hello, I am trying to replicate this but on the new bootscore 6 (which I really like, well done guys!), but I can't find a way around.

Any chance you would be so kind to help me please?

**Reply by crftwrk** [reply]
Please open for each question a new discussion to keep things clear. You can choose icon in the block settings:

![screen](https://github.com/bootscore/bootscore/assets/51531217/8eb4045c-df3c-4f1d-8ec8-5722c75b6b96)

**Comment by balrogmedia** [comment]
What I mean is that I would like to have only the search icon on the navbar, like here:
![Screenshot 2024-05-27 at 11 28 03](https://github.com/bootscore/bootscore/assets/98542232/e88dba48-d2f6-4621-9fe4-8b5008b1dda8)
instead of having the search filed in the navbar

![Screenshot 2024-05-27 at 11 28 43](https://github.com/bootscore/bootscore/assets/98542232/7d47a25d-9b6f-4ee4-9a85-7f415289bf7d)

**Comment by crftwrk** [comment]
Go to bootscores root folder to `template-parts/header/` and override `actions.php` and `collapse-search.php` by copy them to your child using the same folder structre `template-parts/header/[file].php`.

Now replace entire content:

#### actions.php

```php
<?php

/**
 * Template part for displaying the header-actions
 *
 * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
 *
 * @package Bootscore
 * @version 6.0.0
 */


// Exit if accessed directly
defined('ABSPATH') || exit;

?>

<!-- Search toggler -->
<?php if (is_active_sidebar('top-nav-search')) : ?>
  <button class="<?= apply_filters('bootscore/class/header/button', 'btn btn-outline-secondary', 'search-toggler'); ?> ms-1 ms-md-2 search-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-search" aria-expanded="false" aria-controls="collapse-search">
    <i class="fa-solid fa-magnifying-glass"></i><span class="visually-hidden-focusable">Search</span>
  </button>
<?php endif; ?>
```

#### collapse-search.php


```php
<?php

/**
 * Template part for displaying the top-nav searchform collapse widget
 *
 * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
 *
 * @package Bootscore
 * @version 6.0.0
 */


// Exit if accessed directly
defined('ABSPATH') || exit;

?>


<!-- Collapse Search -->
<?php if (is_active_sidebar('top-nav-search')) : ?>
  <div class="collapse <?= apply_filters('bootscore/class/header/collapse', 'bg-body-tertiary position-absolute start-0 end-0'); ?>" id="collapse-search">
    <div class="<?= apply_filters('bootscore/class/container', 'container', 'collapse-search'); ?> pb-2">
      <?php dynamic_sidebar('top-nav-search'); ?>
    </div>
  </div>
<?php endif; ?>

```

Solved?

**Reply by balrogmedia** [reply]
yes, thank you very much!
